docs/conf.py

```diff
@@ -39,7 +39,7 @@
     'sphinx.ext.autodoc',
     'sphinx.ext.intersphinx',
     'sphinx.ext.viewcode',
-    'alabaster'
+    'alabaster',
 ]
```

Comment:
Why trailing comment?

Reply:
so you get single line diff when you add new item to a list.